code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -277,15 +278,14 @@
 			stat |= NOPOWER
 
 
-/obj/machinery/chem_master/attackby(var/obj/item/weapon/B as obj, var/mob/user as mob)
+/obj/machinery/chem_master/attackby(var/obj/item/B as obj, var/mob/user as mob)

```

Comment:
Forgot to revert this line when reverting finite bottles, anyone think it'll mater?